src/test/java/com/sfuentes/TheaterTest.java

```diff
@@ -0,0 +1,118 @@
+package com.sfuentes;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class TheaterTest {
+
+  @Test
+  void getCategoryByDiscount_categoryOneAgeRangeFiveToFourteen_thirtyFiveDiscount() {
+    ArrayList<Integer> ages = new ArrayList<>(Arrays.asList(5, 6, 7, 8, 9, 10, 11, 12, 13, 14));
```

Comment:
To avoid these long lists of integers you can create another method to generate them, something like this:
  List<Integer> range(int start, int end){
    return IntStream.rangeClosed(start, end)
        .boxed().collect(Collectors.toList());
  }

then you can just call it like this:
 ArrayList<Integer> ages = range(5,14);

Reply:
fixed

Reply:
fix